### ISSUE TITLE
zip module does not have close method

### ISSUE THREAD

**Issue by kriszyp** [tlrobinson/narwhal#73]
The zip module does not have a close method, and so calling unzip leaves the file in an open/locked state. Fix is here:
http://github.com/kriszyp/narwhal/commit/27b49bfe9c5372f820f3130d302606b797f8c24a

**Comment by kriskowal** [comment]
Landed in kriskowal/master kriskowal/refactor.  I cherry-picked; you're tracking cadorn's stuff, right?  Does it work for you?

**Comment by kriszyp** [comment]
I have pulled in cadorn's master, and unzip still works with it. Or are you saying that cadorn's master should fix unzip to close? I don't see anything in cadorn's zip.js that would close.

**Comment by kriskowal** [comment]
No, unrelated; tusk does not work for me on cadorn/experimental and was wondering if anyone else had succeeded in using it.  So it would seem; I'll have to look deeper.
